### DISCUSSION TITLE
Chrome web store rejected my extension: "Request but do not use the following permissions: storage"

### DISCUSSION THREAD

**Discussion by Aaronliu2016** [wxt-dev/wxt#548] [Q&A] [answered]
Hello everyone,
I am using wxt.dev for the first time, and it is very amazing. It greatly improves my development efficiency. However, I encountered some problems when trying to submit an extension to the Chrome Web Store.
My extension was rejected after several days of review, and the reason given was:

> Request but do not use the following permissions:
>    storage

I don't understand why.My extension needs to store data in the browser's storage. I have used storage.defineItem<> in the code and it runs fine. Try removing the storage permission in the manifest, it clearly doesn't work: 

> Error: You must add the 'storage' permission to your manifest to use 'wxt/storage'.

But why did the reviewer say that I didn't use storage permission?

**Comment by aklinker1** [comment, marked as answer]
Huh, they're just wrong. Either an automated tool or the reviewer must have messed up. I've released an extension that uses the `wxt/storage` module just fine... I can't remember, but I think there's a way to respond to rejections? Link them this comment.

WXT doesn't use `chrome.storage.local` directly, so if they're doing a simple search for something like that, it won't show up. Instead, it uses this code to get the storage area based on the prefix: 

https://github.com/wxt-dev/wxt/blob/53c63b45568fe875f11d1c0cfd9234bd808bf5de/src/storage.ts#L364

I could explicitly use `browser.storage.local` in the code of `browser.storage[storageArea]`, but I'd rather not make that change if it's not necessary...

**Reply by Aaronliu2016** [reply]
Lol, you're right. I have emailed the support team and they admitted that it was a mistake. Already re-submitted, hope everything goes right this time.
Thanks!

**Reply by aklinker1** [reply]
Sorry about that! The Chrome Web Store usually does a good job of reviewing extensions. If other people experience the same problem, I'll update the code so they can find the usage easier.
